src/applications/vre/28-1900/wizard/pages/veteran/01-noHonorableDischarge.jsx

```diff
@@ -5,7 +5,10 @@ import { recordNotificationEvent, fireLinkClickEvent } from '../helpers';
 const NoHonorableDischarge = () => {
   recordNotificationEvent('ineligibility - received a dishonorable discharge');
   return (
-    <div className="feature vads-u-background-color--gray-lightest">
+    <div
+      className="feature vads-u-background-color--gray-lightest"
+      aria-live="polite"
```

Comment:
We may want to include `aria-atomic="true"` just in case so the full message is announced instead of just what's in the {' '} 🤔

Reply:
Added aria-atomic [in this commit](https://github.com/department-of-veterans-affairs/vets-website/pull/16564/commits/563d7c65736e903d398f0553f22549e50239485d)